### DISCUSSION TITLE
Should train platforms be asked for name?

### DISCUSSION THREAD

**Discussion by tommycrock** [streetcomplete/StreetComplete#5145] [Q&A] [answered]
A recent(ish) change introduced the bus stop/platform quest for ways. I can see the benefit of this with train platforms for quests like bin and shelter, but am unsure about names. 

There seems to be differing international practice for platform names, e.g. Germany uses the name of the station, in France and Poland it's often Platform #, with 'platform' in the local language and ref=# where used. In the UK, most train platforms are only tagged with ref, and that seems to be the main preference in discussion, although at least one contributor prefers Platform #. 

Should this be switched off for train platforms (but preferably still on for trams), at least in the UK? Or is it time the UK started tagging train platform names (in which case a hint for the format might help)? Or at least SC should not act surprised if I claim there is no name. Thanks 👍

**Comment by burrscurr** [comment]
I recently mapped some `railway=platform_edge`s on railway platforms in Germany and rarely came across meaningful platform names. It is either the station name or station name followed by the platform number. In both cases, that's not really what a `name` tag is for (merely a description rather than an actual name). So I agree with disabling the name quest for railway platforms.

**Comment by burrscurr** [comment, marked as answer]
This should be resolved now, have a look at #5215 for details.
